### ISSUE TITLE
listen sound when/before loading file

### ISSUE THREAD

**Issue by proteinkyn** [sonejostudios/superboucle#50]
Hi
I don't known if it possible but it will be great. For example, like hydrogen.

Thank you very much

**Comment by IARI** [comment]
Can you describe what you mean? (probably @sonejostudios knows, I have hardly any memories of Hydrogen)
What would be playing and when?
My interpretation would be an audio-preview in the "open song" file dialogue (it could for instance be playing back the initial scene of the song)

**Comment by sonejostudios** [comment]
Do you think listen a sound file before loading it into a clip?
Since you can drag and drop sound files into clips, you can easily listen them in an external player or in the file browser before loading them into your project :)

**Comment by proteinkyn** [comment]
Thank you for your answers.
I can't drag and drop. I use ubuntu (KXstudio) and I don't know why listen music when the mouse is on the file in nautilus!!
This is a example of the door for browse sounds files in hydrogen. You can listen the file before loading. And you can see it. But it's not necessary. 
Thank you very much.
![hydrogen_browser-file](https://cloud.githubusercontent.com/assets/8607505/15255014/f5798710-1939-11e6-8152-38e79b1cb126.png)

**Comment by sonejostudios** [comment]
which version of superboucle are you using? our latest?
@IARI do you think the this drag n drop problem is related to kde? on mate and gnome it works, though...

**Comment by IARI** [comment]
To include something like this would require to implement a custom file-loading dialog - we are using some standard-qt dialog as of now. I think we don't have the time to implement it any time soon, since there are other things that have to be done first.

Would fixing the drag-and-drop issue be a temporary solution for your need?
Regarding that @proteinkyn  I would ask you to open a new issue. Please describe everything as detailed as possible, what happens when you try to drag an drop samples from your file browser into superboucle.

**Comment by proteinkyn** [comment]
Hi
Thank you very much. Sorry for my answer in late.
1- I use gnome-sushi since 3 days and it's enough for listening file 
2- I didn't understand the drag & drop. I thought that the drag & drop was between the file browser (nautilus) and the window "add clip". But if I drag & drop directly in the box of the loop that's good.
It's easier to load file!!!
Sorry for the inconvenience.
Thank you very much.

**Comment by IARI** [comment]
No problem - @sonejostudios  it could be a hint for us still. we might want to improve the clarity to the user how drag and drop works.
(Unless we want to redesign the check-gui even more)
